src/validatePrTitle.test.js

```diff
@@ -69,6 +79,16 @@ describe('defined scopes', () => {
     );
   });
 
+  it('throws when an unknown scope is detected within multiple scopes', async () => {
+    await expect(
+      validatePrTitle('fix(CORE,e2e,foo,bar): Bar', {
+        scopes: ['foo', '[A-Z]+']
+      })
+    ).rejects.toThrow(
+      'Unknown scopes "e2e,bar" found in pull request title "fix(CORE,e2e,foo,bar): Bar". Use one of the available scopes: foo, [A-Z]+.'
```

Comment:
Maybe we could phrase the part "Use one of the available scopes" a bit differently now to match the regex case better.

Reply:
Maybe something like: `Scope must match one of: <list here>`?

Reply:
Yep, sounds great!

Reply:
Changed! Was also used in the required scope check, so updated that as well :)